src/martialarts.cpp

```
@@ -91,12 +91,17 @@ class ma_weapon_damage_reader : public generic_typed_reader<ma_weapon_damage_rea
 
         std::pair<damage_type, int> get_next( JsonIn &jin ) const {
             JsonObject jo = jin.get_object();
-            const damage_type dt = dt_map.find( jo.get_string( "type" ) )->second;
+            std::string type = jo.get_string( "type" );
+            std::string s = dt_map.find( type ) != dt_map.end() ? type : "true";
```

It's better to throw a JSON error instead if `type` is not found in `dt_map`.